dnf-behave-tests/features/command-aliases.feature

```diff
@@ -0,0 +1,105 @@
+Feature: Tests for command aliases availability
+
+
+Scenario: "nonexistent" is not an alias for any dnf command
+   When I execute dnf with args "nonexistent -h"
+   Then the exit code is 1
+    And stderr contains "No such command: nonexistent"
+
+
+Scenario Outline: "<alias>" is an alias for "<command>"
+   When I execute dnf with args "<alias> -h"
+   Then the exit code is 0
+    And stdout contains "usage: {context.dnf.dnf_command} <command>"
```

Comment:
Hmm, the `{context.dnf.dnf_command}` doesn't work if you have a wrapper script around dnf, which e.g. I do :slightly_frowning_face:. So that isn't great...

Reply:
Well, this is unpleasant. It's not so much of a problem for this PR as it is for https://github.com/rpm-software-management/ci-dnf-stack/pull/612 .

I don't see any way of how to get the command printed by help other than running the help. However, I could remove the test 'I execute dnf with args' and leave only the specific 'I execute "dnf" with args' and other variants. I think it would be sufficient. In this PR, I could replace it with ".*", because it's not important part of this test. What do you think?

Reply:
Yeah, for #612 the 'I execute "dnf" with args' tests should be sufficient I think.

I'm not fond of the `.*` but it's probably the best option.

I could live with the test failing and I could also work around the issue by e.g. setting up a symlink to dnf that would have the same name as the wrapper script :slightly_smiling_face: but it's a bit cumbersome.

Not sure if others use such wrapper script, but you (AFAIK) need it to run the ci stack against git and I find it very useful, it's the only way I test majority of the time nowv (because it's so fast).

Reply:
Ok, I removed the test that uses 'I execute dnf with args' from the https://github.com/rpm-software-management/ci-dnf-stack/pull/612 and replaced the command with ".*" here.